store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/AddGraph.java

```diff
@@ -89,6 +89,7 @@ public AddGraph shallowClone() throws CloneFailedException {
                 .parentSchemaIds(parentSchemaIds)
                 .parentPropertiesId(parentPropertiesId)
                 .options(options)
+                .setGraphAuths(graphAuths.toArray(new String[graphAuths.size()]))
```

Comment:
This logic now means the test FederatedStoreTest.shouldAddGraphIdWithAuths fails.